backend/app/settings.py

```diff
@@ -0,0 +1,39 @@
+from functools import lru_cache
+from typing import Optional
+
+from pydantic import HttpUrl
+from pydantic_settings import BaseSettings, SettingsConfigDict
+
+
+class Settings(BaseSettings):
+    model_config = SettingsConfigDict(env_file=".env")
```

Comment:
This will be overwritten by the same ENV variables being populated right?

Reply:
yes